src/components/join-list/styles.css

```diff
@@ -0,0 +1,3 @@
+.errorAuth {
+  color: red;
```

Comment:
I like this class. On another hand, for accessibility issues and people color-blind, please ["Don’t rely solely on color"](https://www.invisionapp.com/inside-design/color-accessibility-product-design/).

Suggestion: please 🙏 , if you can consider that, you might add an icon. 😄

Reply:
@susird Hi Susana! I saw that this class is being used in the Welcome screen. Does it make sense for you to improve your PR so that we don't generate conflicts?